.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,75 @@
+name: UnitTests
+
+on:
+    pull_request:
+    push:
+
+jobs:
+    tests:
+        name: "PHP ${{ matrix.php }} + Composer ${{ matrix.dependency }} + Symfony ${{ matrix.symfony }}"
+        runs-on: ubuntu-latest
+
+        strategy:
+            fail-fast: true
+            matrix:
+                include:
+                    -   php: '5.5'
+                        dependency: lowest
+                        coverage: xdebug
```

Comment:
what is the benefit of using both, xdebug and pcov? for us, we care that PHPUnit is able to generate the coverage, we don't need to check that PHPUnit is able to generate it with different drivers. let's use only pcov

Reply:
pcov working much faster, but pcov allowed only for PHP 7.2 and higher

Reply:
I believe we are ok to run tests on each supported version and do actual upload to coveralls API on one of them only.

Reply:
Done